tools/zuul-projects-checks.py

```diff
@@ -170,18 +173,54 @@ def check_only_boilerplate():
     return errors
 
 
+def check_project_templates():
```

Comment:
lets add something like other functions... eg:

```
print("\nChecking that every project has entries")
print("======================")
```

Then check errors, and output some instructions